price_updater/.env.example

```diff
@@ -0,0 +1,17 @@
+# Sepolia
+MAINNET_RPC_URL=https://example.com
+ARBITRUM_RPC_URL=https://example.com
+PRIVATE_KEY=0x2......
+TARGET_CHAIN=10003
+TARGET_ADDRESS=0x744836a91f5151c6ef730eb7e07c232997debaaa
+PRICE_FEED=0x38211b7ed71fa91fd65b69b6bf2c02efe60fc35b
+PRICE_FEED_SENDER=0xe572a8631a49ec4c334812bb692beecf934ac4e9
+
+# Mainnet
+MAINNET_RPC_URL=https://example.com
```

Comment:
Lets move MAINNET_RPC_URL, PRIVATE_KEY, PRICE_FEED_SENDER to top as they must be provided for any network